crates/wasi-provider/src/lib.rs

```diff
@@ -107,51 +235,30 @@ impl<S: Store + Send + Sync> Provider for WasiProvider<S> {
         let mut modules = self.store.fetch_pod_modules(&pod).await?;
         let client = kube::Client::new(self.kubeconfig.clone());
         let volumes = Ref::volumes_from_pod(&self.volume_path, &pod, &client).await?;
+        info!("Running init containers for pod {:?}", pod_name);
+        for container in pod.init_containers() {
+            self.run_one_container_to_completion(
+                container,
+                &pod,
+                &client,
+                &mut modules,
+                &volumes,
+                &mut container_handles,
+            )
+            .await?;
+        }
+        info!("Finished running init containers for pod {:?}", pod_name);
```

Comment:
We should probably put a debug log in the loop saying which init container we are running right before it starts and right after it finishes

Reply:
The start message is already there in `start_container` (fka `run_one_container`).  Added an end message to `run_container_to_completion`.